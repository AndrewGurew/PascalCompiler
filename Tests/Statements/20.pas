begin
foo(1,1+1,b+a);
end