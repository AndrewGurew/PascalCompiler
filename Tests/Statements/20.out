Main block
 ⎬ Procedure or Function Call foo with params:
  ⎬1
  ⎬+
   ⎬1
   ⎬1
  ⎬+
   ⎬b
   ⎬a
End of Main block
